src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public abstract class ColorSupplier extends Figure {

```

Comment:
Class names should not be verbs. In this case, `ColorSupplier` is acceptable because it acts as a noun, suggesting it supplies colors.